Removed istio from kf install

Klaven opened the pull request:
removed istio from the base kubeflow install.
Dont merge yet. it's a WIP.
istio should be installed following standard SUSE documentation

**Which issue is resolved by this Pull Request:**
Resolves #

**Description of your changes:**


**Checklist:**
- [ ] Unit tests have been rebuilt: 
    1. `cd manifests/tests`
    2. `make generate-changed-only`
    3. `make test`

Commit:
removed istio from kf install

removed istio from the base kubeflow install.

istio should be installed following standard SUSE documentation

cmurphy commented:
What's the status of this? Could you update the install instructions to account for the need to install istio by hand?

cmurphy committed:
Add userid-header parameter to profiles (#1700)

Without this commit, the profiles controller will have a "userid-header"
parameter of "" set, which cause the Profile CRD to create an Istio
ServiceRoleBinding with an incorrect subject "request.headers[]:
<profile username>" which will never match anything and cause the user
to not be able to connect to their Jupyter notebooks. This commit
ensures the profiles controller is configured correctly to fix this
problem.

cmurphy commented:
Added a commit to fix https://github.com/SUSE/avant-garde/issues/1700 , with this I think Istio 1.5 works with kubeflow.
